OpenNMT/onmt/Models.py

```diff
@@ -29,10 +29,9 @@ def __init__(self, opt, dicts):
             self.word_lut.weight.copy_(pretrained)
 
     def forward(self, input, hidden=None):
-        emb = self.word_lut(input)
-
+        batch_size = input.size(0) # batch first for multi-gpu compatibility
+        emb = self.word_lut(input).transpose(0, 1)
```

Comment:
Before this PR: Epoch  1,    50/  156 batches; perplexity: 169995.82; 2826 tokens/s

```Epoch  1,   100/  156 batches; perplexity: 29357.66; 3026 tokens/s
Epoch  1,   150/  156 batches; perplexity: 7109.44; 3126 tokens/s
Train perplexity: 27737.7
Validation perplexity: 2487.7

Epoch  2,    50/  156 batches; perplexity: 3122.17; 2974 tokens/s
Epoch  2,   100/  156 batches; perplexity: 2255.79; 3118 tokens/s
Epoch  2,   150/  156 batches; perplexity: 1812.20; 2968 tokens/s
Train perplexity: 2311.4
Validation perplexity: 1759
```

and after this PR:

```
Epoch  1,    50/  156 batches; perplexity: 1109740.55; 3038 tokens/s;     26 s elapsed
Epoch  1,   100/  156 batches; perplexity: 14303.51; 3002 tokens/s;     49 s elapsed
Epoch  1,   150/  156 batches; perplexity: 4059.63; 2937 tokens/s;     72 s elapsed
Train perplexity: 43706.5
Validation perplexity: 3200.05

Epoch  2,    50/  156 batches; perplexity: 2723.29; 2962 tokens/s;    103 s elapsed
Epoch  2,   100/  156 batches; perplexity: 1981.58; 3061 tokens/s;    128 s elapsed
Epoch  2,   150/  156 batches; perplexity: 1816.14; 2977 tokens/s;    153 s elapsed
Train perplexity: 2111.29
Validation perplexity: 2029.71
```